specification/cognitiveservices/data-plane/TextAnalytics/preview/v3.1-preview.4/TextAnalytics.json

```diff
@@ -149,6 +149,11 @@
             "$ref": ".//examples//SuccessfulAnalyzeStatusRequest.json"
           }
         },
+        "x-ms-pageable": {
+          "operationName": "AnalyzeStatusNextPage",
+          "nextLinkName": "nextLink",
+          "itemName": "values"
```

Comment:
There is no property on the response object called `values`, I think it should be `tasks` in this case